rust/src/composepost.rs

```diff
@@ -1201,7 +1201,18 @@ fn workaround_selinux_cross_labeling_recurse(
 /// This is the nearly the last code executed before we run `ostree commit`.
 pub fn compose_postprocess_final(rootfs_dfd: i32, _treefile: &Treefile) -> CxxResult<()> {
     let rootfs = unsafe { &crate::ffiutil::ffi_dirfd(rootfs_dfd)? };
-
+    if std::process::Command::new("add-determinism").status().expect("Failed to find add-determinism on system.").success() {
+         // add-determinism --handler pyc-zero-mtime
+        let r = std::process::Command::new("add-determinism")
+        .arg("--handler")
+        .arg("pyc-zero-mtime")
+        .arg("/usr")
```

Comment:
This won't do what you expect or want. In an rpm-ostree build, the `/usr` is whatever happens to be the build environment, not the target.

In rpm-ostree we try hard to use fd-relative accesses generally via cap-std.

To do so with an external process, a typical pattern is to use e.g. [cwd_dir](https://docs.rs/cap-std-ext/latest/cap_std_ext/cmdext/trait.CapStdExtCommandExt.html#tymethod.cwd_dir) and then pass `.` to the child process as the target.